i18n/en/docusaurus-plugin-content-docs/current/get-started/overview.md

```diff
@@ -0,0 +1,172 @@
+---
+sidebar_position: 1
+---
+
+# Overview
+
+:::caution
+
+Only basic information on the methodology is presented here
+
+For a more competent application, it is worth getting acquainted with each concept in more detail in the corresponding section of the documentation
+
+:::
+
+## Is it right for me?
+
+FSD can be implemented in projects and teams of any size, but there are a few things to keep in mind:
+
+- This methodology is for front-end projects only. If you're looking for a back-end architecture, consider [Clean Architecture][refs-clean-architecture].
+- A very simple app of a single page might not need the benefits of FSD and suffer from the overhead. However, FSD promotes a nice way of thinking, so feel free to use on the tiniest projects if you want.
+- A huge app, the size of the Google Cloud admin dashboard, will require a custom architecture. It could still be based on FSD, by the way.
+
+FSD doesn't enforce a particular programming language, UI framework or state manager — bring your own or see some [examples][refs-examples].
+
+If you have an existing project, fear not — FSD can be adopted incrementally. Just make sure that your team is **in&nbsp;pain** from the current architecture, otherwise a switch might not be worth it. <!-- For migration guidance, see the Migration section. TODO: add the migration section and link to it here -->
+
+## Basics
+
+In FSD, a [project consists][refs-splitting] of <mark>layers</mark>, each layer is made up of <mark>slices</mark> and each slice is made up of <mark>segments</mark>. 
```

Comment:
> 4. [minor] Not a fan of those bright blue rectangles around the terms

![image](https://user-images.githubusercontent.com/42924400/188972150-36be31e0-4366-45d4-99bd-f970eaa6b4a3.png)

I tried different variants, but highlighting by mark at this place - looked good for me
(with comparing of default styles, _italic_, `code` or **bold**)

Also as you said - it need for accent basic abstractions of FSD
And basic content visual splitting for easy reading
(i'm not sure that it so "hurts" 🤷‍♂️ )

@feature-sliced/core What do u think?
(especially - @unordinarity @Rin-Akaia-eth)

Reply:
> (i'm not sure that it so "hurts" 🤷‍♂️ )

UPD: I also tried to overdo with it, of course

Its why I dont highlight another parts with abstractions and concepts 😏

Reply:
UPD:UPD: But as compromise - i could omit highlighting for "detailed description"

I mean

```md
<!-- with highlighting -->
In FSD, a [project consists][refs-splitting] of <mark>layers</mark>, each layer is made up of <mark>slices</mark> and each slice is made up of <mark>segments</mark>.
```

```md
<!-- without highlighting -->
The **layers** are standardized across all projects and vertically arranged. ...//

Then there are **slices**, which partition the code by business domain. ..//

Each slice, in turn, consists of **segments** ...//
```

Reply:
> it need for accent basic abstractions of FSD

But is there a real problem being solved here? I don't think there are any complaints that the users are struggling to understand that FSD is really about these three things, especially when they are doubled with a visual diagram which represents the things and nothing more. I think we don't need more highlighting here, italics should work fine

Reply:
> But is there a real problem being solved here? I don't think there are any complaints that the users are struggling to understand that FSD is really about these three things, especially when they are doubled with a visual diagram which represents the things and nothing more. 

For example

<img width="559" alt="image" src="https://user-images.githubusercontent.com/42924400/189478129-5a80f326-42e8-4420-8b67-874a1bbddbc6.png">

Reply:
>  I think we don't need more highlighting here, italics should work fine

@Rin-Akaia-eth @unordinarity @Imperyall

Reply:
I like these rectangles, but I definitely would like to use another color for them, something much more neutral

Reply:
Then we could use `code` snippets - they have same styles but with grayscale

But it intersect with layers examples
(`shared` `entities`... and `layers` `slices` `segmebnts`)

Also can suggest to save "blue rectangle" but with less saturation (more gray, but not absolute grayscale)

@RinAlaiaEth @illright what do u think?

Reply:
I'm personally against any background for these. I could accept colored text (but we need to be careful not to make them look like links), but the background adds too much excess emphasis, imo

Reply:
@azinit, @illright, I just realized that I would actually make every methodology concept mention to be a link to the page about it ( wikification? )

Reply:
It will also resolve a highlighting problem

Reply:
I'd object to having them as links as well. It is indeed wikification, and what works for the wiki doesn't necessarily work for the introduction page. Since the start of my work on the docs I've been on a quest to decrease the amount of links in the introduction to an absolute minimum

Reply:
@illright why do you think it's a problem?

Reply:
fixed